packages/shared/src/components/cards/v1/ActionButtons.tsx

```diff
@@ -29,7 +29,7 @@ function ShareButton(props: ShareButtonProps) {
   return (
     <SimpleTooltip content="Share post">
       <Button
-        className="ml-2"
+        className="pointer-events-auto ml-2"
```

Comment:
Do we really need to define these?
Feels a bit double and we could easily forget elements.